ubuntu/debian/tests/build

```diff
@@ -24,8 +24,8 @@ int main(int argc, char **argv)
 }
 EOF
 
-g++ -o igntest igntest.c `pkg-config --cflags --libs ignition-common5`
+g++ -o gztest gztest.c `pkg-config --cflags --libs gz-common5`
```

Comment:
I believe this line will require https://github.com/gazebosim/gz-common/pull/352